R/git.R

```diff
@@ -28,6 +28,9 @@ use_git <- function(message = "Initial commit") {
     if (yep(commit_consent_msg)) {
       done("Adding files and committing")
       git2r::add(r, paths)
+      if (length(paths) > 0) {
```

Comment:
I feel `git_uncommitted()` (in the condition for this block) implies that `paths` has non-zero length. Have you seen something to the contrary?

Reply:
This PR part was for https://github.com/r-lib/usethis/issues/128

Reply:
If there are two different issues, let's split into two PRs then.

Reply:
LIttle swamped now, will within next week or 2.

Reply:
OK thanks!